metricbeat/module/jolokia/jmx/data.go

```diff
@@ -149,7 +151,10 @@ func parseResponseEntry(
 ) error {
 	field, exists := mapping.Get(requestMbeanName, attributeName)
 	if !exists {
-		return errors.Errorf("metric key '%v' for mbean '%s' not found in mapping (%+v)", attributeName, requestMbeanName, mapping)
+		// This shouldn't ever happen, if it does it is probably that some of our
+		// assumptions when building the request and the mapping is wrong.
+		logp.Debug("jolokia.jmx", fmt.Sprintf("mapping: %+v", mapping))
```

Comment:
you don't need the `fmt.Sprintf` part here, you can write:

```
logp.Debug("jolokia.jmx", "mapping: %+v", mapping)
```

Reply:
Or you can use `Debugw` with a the structured logger with a "jolokia" context.

Reply:
Much better without `Sprintf` yes, thanks :)